conda/cli/install.py

```diff
@@ -264,13 +266,19 @@ def install(args, parser, command='install'):
                     force_reinstall=context.force_reinstall or context.force,
                     should_retry_solve=(_should_retry_unfrozen or repodata_fn != repodata_fns[-1]),
                 )
+
+                if (context.prerelease_behavior is PrereleaseBehavior.LIMIT and
+                        not is_last_repodata and
+                        unlink_link_transaction.adds_prereleases):
```

Comment:
I know the rest of the codebase is written with a different style, but as per the new style guide, we are trying to become "darker" little by little :D As a result, can we write this like this?

```suggestion
                if (
                    context.prerelease_behavior is PrereleaseBehavior.LIMIT 
                    and not is_last_repodata
                    and unlink_link_transaction.adds_prereleases
                ):
```